benchmark/t-test.js

```diff
@@ -0,0 +1,130 @@
+'use strict';
+
+// Class for doing statistic test following a Student's t-distribution,
+// with default configuration: mu = 0, alternative = two.sided
+function TTest(leftData, rightData) {
+  this.left = this.createData(leftData);
+  this.right = this.createData(rightData);
+  this.df = this.left.size + this.right.size - 2;
+  this.dfhalf = this.df / 2;
+  this.mean = this.left.mean - this.right.mean;
+  this.commonVariance = ((this.left.size - 1) * this.left.variance +
+                         (this.right.size - 1) * this.right.variance) / this.df;
+  this.se = Math.sqrt(this.commonVariance * (1 / this.left.size + 1 /
+                      this.right.size));
+}
+
+TTest.prototype.createData = function(data) {
+  // Calculate the sum
+  var sum = 0;
+  var i;
+  for (i = 0; i < data.length; i++) {
+    sum += data[i];
+  }
+  // Calculate the mean.
+  var mean = sum / data.length;
```

Comment:
That can only happen if the developer run the compare.js without passing sets, and this case is already covered by the CLI. But we can put a check.